$NetBSD: patch-bq,v 1.4 2005/12/15 21:28:48 wennmach Exp $

But I do need HAVE_KRB4. I want it, want it, want it!

--- rxkad/rxk_serv.c.orig	2005-12-12 17:25:17.000000000 +0100
+++ rxkad/rxk_serv.c	2005-12-12 17:25:28.000000000 +0100
@@ -39,11 +39,7 @@
 #include <krb5.h>
 #endif
 
-#undef HAVE_KRB4
-
 #ifdef HAVE_KRB4
-#include <krb.h>
-#else
 #include <krb4.h>
 #endif
 
